audit-triplestore/src/main/java/org/fcrepo/camel/audit/triplestore/EventRouter.java

```diff
@@ -43,7 +44,9 @@ public void configure() throws Exception {
          */
         from("activemq:{{jms.fcrepoEndpoint}}")
             .routeId("AuditFcrepoRouter")
-            .to("direct:event");
+            .choice()
+            .when(not(header("org.fcrepo.jms.identifier").startsWith(simple("{{audit.container}}"))))
+                .to("direct:event");
```

Comment:
this works, but filter(...) would be simpler

    from("...")
        .routeId("AuditFcrepoRouter")
        .filter(not(header(JmsHeaders.IDENTIFIER).startsWith(simple("..."))))
            .to("direct:event")

Also note that the fcrep-camel component has CONST values for the JMS headers